misc/documentation_styleguide.md

```diff
@@ -53,6 +53,24 @@ This is some text.
 This is some more text.
 ```
 
+## Bullet and numbered items
+
+First level is flushed all the way to the left. A sub level is offset by 4 spaces. The next sub level is offset by 8 spaces. You get the idea!
+
+```
+* The first level bullet point
+    * A second level bullet point
+        * A third level bullet point
+```
+
+## Links
+
+When cross reference a document, there are three kinds: within a document, within a project group, and outside a project group. The `[displayed text]` should be to the point and within a pair of square brackets. The displayed text must be followed by `(the link)` enclosed in a pair parenthesis. Follow the formats below to ensure link connection.
+
+* Reference within the same document: the link is proceeded with a # - [General rules]`(#General rules)`.
+* Reference within a project group: the link is proceeded with ../ - [Firefox Focus]`(../products/focus/testing_focus.md)`.
```

Comment:
```
the link is proceeded with ../ - [Firefox Focus]`(../products/focus/testing_focus.md)`.
```

Better:

```
use relative links, instead of absolute links (starting with `/`) or full GitHub URLs. For example, to link to a document in the parent folder, use `[other document](../other.md)`.
```